frontend/src/app/modules/nav/desktop-nav/desktop-nav.component.ts

```diff
@@ -8,6 +8,7 @@ import {ChangeDetectionStrategy, Component, EventEmitter, Input, Output} from '@
 })
 export class DesktopNavComponent {
     @Input() isLoggedIn = false;
+    @Input() gtmContext;
```

Comment:
[could] Maybe more meaningful name for context?

Reply:
Actually, as this is an input prop, this name seems to be a little bit better if we think of it as an external API.